Queens_Problem/Queens_Problem.cpp

```diff
@@ -69,7 +69,7 @@ int N_queen(int n)
             if((is_safe(row,col)) && (board[row][col]!=1))
             {
                 board[row][col] = 1;
-                //recursion;wether we can put the next queen with this arrangment or not
+                //recursion ; whether we can put the next queen with this arrangement or not
                 if(N_queen( n-1 ) == 1)
```

Comment:
Line 72: Remove the space before `;`.

Line 73: spacing is not fixed yet - `if(N_queen( n-1 ) == 1)` should be `if(N_queen(n - 1) == 1)`.

 We never add space before punctuation marks. Neither in english, nor in code.
We never add space just after an opening bracket or just before a closing bracket both in english and code.
We should try to always add a space before and after operators in code. (in english also)